mbed-trace/mbed_trace.h

```diff
@@ -311,20 +321,14 @@ char* mbed_trace_ipv6_prefix(const uint8_t *prefix, uint8_t prefix_len);
 #define mbed_trace_print_function_set(...)  ((void) 0)
 #define mbed_trace_cmdprint_function_set(...)  ((void) 0)
 #define mbed_trace_exclude_filters_set(...) ((void) 0)
-#define mbed_trace_exclude_filters_get(...) ((void) 0)
+#define mbed_trace_exclude_filters_get(...) ((char *) 0)
 #define mbed_trace_include_filters_set(...) ((void) 0)
-#define mbed_trace_include_filters_get(...) ((void) 0)
-
-#define mbed_trace_last(...)                ((void) 0)
+#define mbed_trace_include_filters_get(...) ((char *) 0)
+#define mbed_trace_last(...)                ((char *) 0)
 #define mbed_tracef(...)                    ((void) 0)
-#define mbed_trace_ipv6(...)                ((void) 0)
-#define mbed_trace_array(...)               ((void) 0)
-#define mbed_trace_ipv6_prefix(...)         ((void) 0)
 
-#endif //YOTTA_CFG_MBED_TRACE
+#define mbed_trace_ipv6(...)                dont_use_trace_helpers_outside_trace_calls
```

Comment:
Maybe worth a comment explaining the restriction - they accumulate strings in a buffer that is only flushed by trace calls, and could overflow if used outside them.